<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Quoridor</class>
 <widget class="QMainWindow" name="Quoridor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="baseWidget">
   <layout class="QVBoxLayout" name="baseLayout">
    <item>
     <widget class="QWidget" name="mainMenuWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="mainMenuLayout">
         <item>
          <layout class="QFormLayout" name="boardSizeLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="boardSizeLabel">
             <property name="text">
              <string>Taille du plateau :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="boardSizeInput">
             <property name="minimum">
              <number>5</number>
             </property>
             <property name="maximum">
              <number>19</number>
             </property>
             <property name="singleStep">
              <number>2</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="nbPlayersGroupBox">
           <property name="title">
            <string>Nombre de joueurs</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="players2RadioButton">
              <property name="text">
               <string>2 joueurs</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="players4RadioButton">
              <property name="text">
               <string>4 joueurs</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="typePlayersLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string>Joueur 1</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QRadioButton" name="player1RealRadioButton">
                  <property name="text">
                   <string>Réél</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="player1IARadioButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>IA</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_4">
               <property name="title">
                <string>Joueur 2</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QRadioButton" name="player2RealRadioButton">
                  <property name="text">
                   <string>Réél</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="player2IARadioButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>IA</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QGroupBox" name="player3GroupBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="title">
                <string>Joueur 3</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QRadioButton" name="player3RealRadioButton">
                  <property name="text">
                   <string>Réél</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="player3IARadioButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>IA</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="player4GroupBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="title">
                <string>Joueur 4</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QRadioButton" name="player4RealRadioButton">
                  <property name="text">
                   <string>Réél</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="player4IARadioButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>IA</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="launchGameButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Lancer la partie</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>304</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouvelle_partie"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAide"/>
  </widget>
  <action name="actionNouvelle_partie">
   <property name="text">
    <string>Nouvelle partie</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>players4RadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>player3GroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>players2RadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>player3GroupBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>players4RadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>player4GroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>players2RadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>player4GroupBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
